inactivity.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Maintainer and Repository Inactivity
+nav_order: 6
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Inactivity Policies
+
+As part of the normal lifecycle of a project codebases and maintainers come and go. This document formalizes a Hyperledger wide policy for archiving repositories and moving maintainers to emeritus status when a project does not have an explicit policy of their own
+
+## Maintainer Inactivity
+
+Hyperledger very much appreciates the contributions of all maintainers but removing write privileges is in the interest of an orderly and secure project.
+
+Maintainers who have not contributed to a Hyperledger project for (`exact timeframe to be decided`) 3 / 4 / 6 months will be retired from active maintainer status and moved to an emeritus status. Any permissions to approve pull requests or commit code and any other such privileges associated with maintainer status will be removed.
```

Comment:
3 months sounds too short.
6 months + 3 month extension sounds too long (9 months total).
My suggestion would be to split the difference: a 4 month inactivity timeframe, with option for 2 month extension, resulting in a total period of 6 months.